internal/store/leaf/fsck.go

```diff
@@ -127,6 +127,14 @@ func (s *Store) fsckCheckEntry(ctx context.Context, name string) error {
 	return nil
 }
 
+func fingerprints(ctx context.Context, crypto backend.Crypto, in []string) []string {
+	out := make([]string, 0, len(in))
+	for _, r := range in {
+		out = append(out, crypto.Fingerprint(ctx, r))
+	}
+	return out
```

Comment:
We might want to sort `out` before return, no? 
Are we guaranteed to receive the keys in the same order every time?
Also `itemRecps` might need to be sorted, no?

Reply:
Shouldn't be necessary. The comparison uses maps: https://github.com/dominikschulz/gopass/blob/57bb81f2af87f2b92b96f535461f3aba84eb5404/internal/store/leaf/fsck.go#L138

Reply:
Nice, I should have checked that function too.